

class IssueExists(Exception):
    pass


class IssueCreateError(Exception):
    pass
